### PR TITLE
Bnc Sanity Scripts added

### PR DESCRIPTION
**Pull request by rkaur10**

### CODE REVIEW

File: spec/bnc/_A_Sanity_Bnc.test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    beforeEach(function() {
+        browser.url(domainName + '/');
+    })
+
+    it('Guest Order Placement flow', function() {
+        this.timeout(0);
+        expect(browser.getTitle()).contain('Welcome to Marks');
+        search.searchProductOnMainPage('p22151415');
+
+        search.pdpAtb();
```
Comment (sumitroy2011) [inline spec/bnc/_A_Sanity_Bnc.test.js]: Please explain what this function is doing and try to use more relevant name

File: spec/bnc/_A_Sanity_Bnc.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        miniBag.miniBagCheckout();
+        checkout.guestCheckout();
+
+        checkout.dmsSddSelected();
```
Comment (sumitroy2011) [inline spec/bnc/_A_Sanity_Bnc.test.js:28]: Explain by giving one liner comment about what the below functions are doing

File: spec/bnc/_A_Sanity_Bnc.test.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        checkout.guestEmail();
+        checkout.orderReview();
+        checkout.placeOrder();
+        expect(browser.getTitle()).contain('Order Confirm');
+        browser.isExisting('#logonPassword');
+    });
+
+    it('Checkout Registration and place order for store delivery', function() {
+        this.timeout(0);
+        search.searchProductOnMainPage('22151415');
```
Comment (sumitroy2011) [inline spec/bnc/_A_Sanity_Bnc.test.js:51]: Bifurcate the functions for different pages and then give one liner comment about what these functions are doing.

**Review by sumitroy2011 — CHANGES_REQUESTED** [review]
Please review the comment and made changes accordingly

File: spec/bnc/_A_Sanity_Bnc.test.js (hunk 2 above)
Comment (rkaur10) [reply spec/bnc/_A_Sanity_Bnc.test.js:28]: done

File: spec/bnc/_A_Sanity_Bnc.test.js (hunk 1 above)
Comment (rkaur10) [reply spec/bnc/_A_Sanity_Bnc.test.js]: done

File: spec/bnc/_A_Sanity_Bnc.test.js (hunk 3 above)
Comment (rkaur10) [reply spec/bnc/_A_Sanity_Bnc.test.js:51]: done

File: common/browseFunctions.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 var browseLoc = require("../resources/browseLocator")
 module.exports = {
     searchProductOnMainPage: searchProductOnMainPage,
-    pdpAtb: pdpAtb,
-    WineSubAtb: WineSubAtb
+    pdpAddtoBag: pdpAddtoBag,
```
Comment (sumitroy2011) [inline common/browseFunctions.js]: Use proper Camel Casing as you did for WineSubAdd**T**oBag

**Review by sumitroy2011 — CHANGES_REQUESTED** [review]
Please change the camel casing for Pdp AddToBag

**Review by sumitroy2011 — APPROVED** [review]
Changes looks good. and This has been tested by Raminder on her local machine. Hence merging the changes

**Pull request merged**